File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const occur = {}
        for (let i=0;i<nums.length;i++){
            let req = target - nums[i]
            console.log(req)
            if(req in occur){
                console.log("Yes found")
                return [occur[req],i]
            }
            occur[nums[i]] = i
        }
        return [-1,-1]
    }
}

File: Data Structures & Algorithms/continuous-subarray-sum/submission-0.py
class Solution:
    def checkSubarraySum(self, nums: List[int], k: int) -> bool:
        remainder = { 0: -1 }
        total = 0
        for i in range(len(nums)):
            total+=nums[i]
            req = total%k
            if req not in remainder:
                remainder[req] = i
            elif i - remainder[req]>1:
                return True
        return False